test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -488,50 +551,70 @@ public class PeekDefinitionTests
-            ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
-
-            PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
-            string objectName = "pd_locationTableType";
-            string schemaName = "dbo";
-            string objectType = "Type";
-
-            Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetUserDefinedTableTypeScripts, objectName, schemaName, objectType);
-            Assert.NotNull(locations);
-            Cleanup(locations);
+            string query = @"
+CREATE TYPE [dbo].[pd_locationTableType] AS TABLE(
+	[LocationName] [varchar](50) NULL,
+	[CostRate] [int] NULL
+)
+GO";
+
+            using (SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, query))
+            {
+                // Get live connectionInfo and serverConnection
+                ConnectionInfo connInfo = TestObjects.InitLiveConnectionInfoForDefinition(testDb.DatabaseName);
+                ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
+
+                PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
+                string objectName = "pd_locationTableType";
+                string schemaName = "dbo";
+                string objectType = "Type";
+
+                Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetUserDefinedTableTypeScripts, objectName, schemaName, objectType);
+                Assert.NotNull(locations);
+                Cleanup(locations);
+            }
         }
 
         /// <summary>
         /// Test get definition for a user defined table type object with active connection. Expect non-null locations
         /// </summary>
         [Fact]
-        public async Task GetUserDefinedTableTypeDefinitionWithoutSchemaNameSuccessTest()
+        public void GetUserDefinedTableTypeDefinitionWithoutSchemaNameSuccessTest()
         {
-            // Get live connectionInfo and serverConnection
-            ConnectionInfo connInfo = await TestObjects.InitLiveConnectionInfoForDefinition();
-            ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
-
-            PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
-            string objectName = "pd_locationTableType";
-            string schemaName = null;
-            string objectType = "Type";
-
-            Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetUserDefinedTableTypeScripts, objectName, schemaName, objectType);
-            Assert.NotNull(locations);
-            Cleanup(locations);
+            string query = @"
+CREATE TYPE [dbo].[pd_locationTableType] AS TABLE(
+	[LocationName] [varchar](50) NULL,
+	[CostRate] [int] NULL
+)
+GO";
+
+            using (SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, query))
```

Comment:
This is minor, but given this test pattern is repeated a lot, we could move it to a common method and just pass in the object name, schema name, type, etc. and maybe the function to be called. E.g. `ExecuteAndValidatePeekTest`. That'd cut down a lot on duplicate test code and make additional tests cheap